src/Microsoft.VisualStudio.Web.CodeGeneration.ProjectInfo/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "keyFile": "../../tools/Key.snk",
+    "nowarn": [
+      "CS1591"
+    ],
+    "debugType": "portable",
+    "emitEntryPoint": false
+  },
+  "dependencies": {
+
+    "Microsoft.CodeAnalysis.Workspaces.Common": "1.3.0",
+    "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0",
+    "NuGet.Frameworks": "3.6.0-*"
```

Comment:
It should work. The only reason I had added 3.6.0 was because I was using the MsBuild APIs here which need 3.6

Since this project doesn't take a dependency on MsBuild APIs, it should be ok to downgrade here.